src/huggingface_hub/hf_api.py

```diff
@@ -896,7 +897,11 @@ def model_info(
             else f"{self.endpoint}/api/models/{repo_id}/revision/{revision}"
         )
         headers = {"authorization": f"Bearer {token}"} if token is not None else None
-        r = requests.get(path, headers=headers, timeout=timeout)
+        status_query_param = {"securityStatus": True} if securityStatus else None
+        path += status_query_param
```

Comment:
```suggestion
```
Since we already pass in `status_query_param` to `params`, this isn't needed.

Reply:
Sorry about this !